tools/monitoring/main.go

```diff
@@ -18,11 +18,15 @@ package main
 
 import (
 	"fmt"
+	"github.com/knative/test-infra/tools/monitoring/yaml-handler"
 	"log"
 	"net/http"
 	"os"
 )
 
+// TODO: replace this with permanent url (or a function) that returns the yaml from test-infra master branch
+const yamlURL = "https://raw.githubusercontent.com/steuhs/test-infra-1/7282b8924d8e2f6f68fe3d34b33ec52a804790fb/tools/monitoring/config-getter/sample.yaml"
```

Comment:
can we point to the sample yaml above instead of the one from your branch?

Reply:
When the app is deployed on gke, it need to point to what is stored on github. We don't want to redeploy the app every time we update the yaml. That's why I don't point it to the sample yaml in the same directory

Reply:
yes. When the update the yaml, we should pick up the latest changes when we query github. Why would we need ot redeploy the app?

The URL here points to your personal branch, which it should not. It should point to the sample yaml being added in this PR.